app/submitter/submitter.py

```diff
@@ -16,9 +16,19 @@ def send_message(self, message, queue, tx_id):  # pylint: disable=unused-argumen
 
 
 class RabbitMQSubmitter():
-    def __init__(self, rabbitmq_url, rabbitmq_secondary_url):
-        self.rabbitmq_url = rabbitmq_url
-        self.rabbitmq_secondary_url = rabbitmq_secondary_url
+    def __init__(self, rabbitmq_host, rabbitmq_secondary_host, rabbitmq_port, username=None, password=None):
+        self.rabbitmq_url = "amqp://{username}:{password}@{host}:{port}/%2F".format(username=username,
+                                                                                    password=password,
+                                                                                    host=rabbitmq_host,
+                                                                                    port=rabbitmq_port) \
+            if username and password else "amqp://{host}:{port}/%2F".format(host=rabbitmq_host,
+                                                                            port=rabbitmq_port)
+        self.rabbitmq_secondary_url = "amqp://{username}:{password}@{host}:{port}/%2F".format(username=username,
+                                                                                              password=password,
+                                                                                              host=rabbitmq_secondary_host,
+                                                                                              port=rabbitmq_port) \
+            if username and password else "amqp://{host}:{port}/%2F".format(host=rabbitmq_secondary_host,
+                                                                            port=rabbitmq_port)
```

Comment:
I think this would read more easily if the username and password conditional was moved to the top level, something like:
```
if username and password:
    ...
else:
    ...
```